### DISCUSSION TITLE
Welcome to geospatial Discussions!

### DISCUSSION THREAD

**Discussion by giswqs** [giswqs/geospatial#15] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ekcomputer** [comment]
Hi everyone. I'm obtaining my PhD in Earth Science (🌎 ⚛️) , studying the geography of lakes and wetlands and their greenhouse gas emissions. I recently discovered this package and it has already saved me a LOT of time installing my preferred stack, and includes virtually every package I need 😄. I hope it can be maintained (along with its sibling packages) and evolve along with its environment.

One suggestion I have is to include the sibling packages to [`psal`](https://github.com/pysal/pysal), which is already present. In particular, I have found [`esda`](https://pysal.org/esda/) and [`pointpats`](https://pointpats.readthedocs.io/en/latest/) helpful, and both are on `conda-forge`.

**Reply by giswqs** [reply]
I have been trying to add `pointpats`, but there are some issues that need to be resolved by the pointpats maintainer before it can be added to `geospatial`. See https://github.com/pysal/pointpats/pull/82. I will see if I can add `esda`.

**Reply by ekcomputer** [reply]
Thanks! Some of the sibling packages get installed with `pysal`, but are not included with `libpysal`. Not sure which is the preferred way to install them.

```python
>> import pysal as ps
>> dir(ps.explore)

['__builtins__',
 '__cached__',
 '__doc__',
 '__file__',
 '__loader__',
 '__name__',
 '__package__',
 '__path__',
 '__spec__',
 'esda',
 'giddy',
 'inequality',
 'pointpats',
 'segregation',
 'spaghetti']
```

**Reply by giswqs** [reply]
`esda` is already included in the `geospatial` conda-forge package. `pysal` can't be added to the geospatial pip until the pointpats issue has been resloved https://github.com/pysal/pointpats/issues/80

![image](https://user-images.githubusercontent.com/5016453/186534959-12955543-d619-447a-84b3-da39dbeb4e42.png)
